src/components/ProposalList/ProposalListItem.tsx

```diff
@@ -1,58 +1,55 @@
-import { Avatar, AvatarImage, AvatarFallback } from '@radix-ui/react-avatar';
 import { Label } from '@radix-ui/react-label';
```

Comment:
TODO: Let's leave the Avatar component in here, we can do something with it soon now that Auth is done
SUGGESTION:
```suggestion
import { Avatar, AvatarImage, AvatarFallback } from '@radix-ui/react-avatar';
import { Label } from '@radix-ui/react-label';
```

Reply:
Quick question, for the avatar. That would be for the person who submitted the proposal and not for the current `auth` user? That would get returned as part of the API response right?

Reply:
PRAISE: Good catch! Can't really display an avatar if we don't get one back in the response schema for that item, can we?

QUESTION: @zenlex Think you could have the endpoint return the author's current avatar as well? If it's simpler I guess we could store the Avatar URL at time of submission, but then it wouldn't update when the user updates it. If need be we can backlog the Avatar component on proposal listings.